samples/TupleToJSONConvert/Makefile

```diff
@@ -1,20 +1,20 @@
-# Copyright (C) 2011,2013,2014, International Business Machines Corporation. 
+# Copyright (C) 2011,2013,2016, International Business Machines Corporation. 
 # All Rights Reserved.
 
-.PHONY: all standalone clean 
+.PHONY: all build clean 
 
-STREAMS_MSG_TOOLKIT ?=../../com.ibm.streamsx.json
-SPLC_FLAGS ?= -a -t $(STREAMS_MSG_TOOLKIT)
+STREAMS_JSON_TOOLKIT ?=../../com.ibm.streamsx.json:$(STREAMS_INSTALL)/toolkits/com.ibm.streamsx.json
```

Comment:
But in distributed, people can still see output in console output?

Perhaps we can enhance the Makefile in the future to use different build command depending on what version of Streams is being used?